queries/typst/injections.scm

```diff
@@ -0,0 +1,7 @@
+(raw_blck
+  (blob) @injection.content)
```

Comment:
This pattern is obsolete with the one below; just remove it. (And add an injection for comments.)